ui/src/app/components/gke-app-configuration-panel.spec.tsx

```diff
@@ -142,84 +147,21 @@ describe(GKEAppConfigurationPanel.name, () => {
 
     expect(notificationStore.get()).toBeNull();
 
-    const wrapper = createWrapper();
-    await waitOneTickAndUpdate(wrapper);
-
-    expect(notificationStore.get()).toBeNull();
-  });
-
-  it('should pass the relevant user app to the specific app component when the API call succeeds', async () => {
```

Comment:
these 3 tests don't fit our new testing paradigm: they test React component internals instead of rendered UI.

Please let me know if you think it's worthwhile to create new equivalent tests for these.